llama_stack/providers/remote/vector_io/qdrant/qdrant.py

```diff
@@ -132,7 +132,50 @@ async def query_keyword(
         k: int,
         score_threshold: float,
     ) -> QueryChunksResponse:
-        raise NotImplementedError("Keyword search is not supported in Qdrant")
+        """Perform keyword search using Qdrant's payload filtering capabilities."""
+        try:
+            results = (
+                await self.client.query_points(
+                    collection_name=self.collection_name,
+                    query_filter=models.Filter(
```

Comment:
nice, looks like this is the appropriate way to handle things https://qdrant.tech/documentation/concepts/filtering/#full-text-match